inbound/trojan.go

```diff
@@ -45,10 +46,14 @@ func NewTrojan(ctx context.Context, router adapter.Router, logger log.ContextLog
 		users: options.Users,
 	}
 	var fallbackHandler N.TCPConnectionHandler
-	if options.Fallback != nil && options.Fallback.Server != "" {
-		inbound.fallbackAddr = options.Fallback.Build()
-		fallbackHandler = adapter.NewUpstreamContextHandler(inbound.fallbackConnection, nil, nil)
+	fallbackAddrs := make(map[string]M.Socksaddr)
+	for _, fallback := range options.Fallbacks {
+		if fallback.Server != "" {
+			fallbackAddrs[strings.ToLower(fallback.Alpn)] = fallback.Build()
+		}
 	}
+	inbound.fallbackAddrs = fallbackAddrs
+	fallbackHandler = adapter.NewUpstreamContextHandler(inbound.fallbackConnection, nil, nil)
```

Comment:
Why is fallback always enabled? you changed its behavior.

Reply:
should check fallbackAddrs > 0?

Reply:
Yes